src/Console/Terminal.php

```diff
@@ -55,7 +55,7 @@ private static function initDimensions(): void
         $consoleMode = self::getConsoleMode();
 
         if ('\\' === \DIRECTORY_SEPARATOR) {
-            if (\preg_match('#^(\d+)x(\d+)(?: \((\d+)x(\d+)\))?$#', \trim(\getenv('ANSICON')), $matches)) {
+            if (is_string(\getenv('ANSICON')) && \preg_match('#^(\d+)x(\d+)(?: \((\d+)x(\d+)\))?$#', \trim(\getenv('ANSICON')), $matches)) {
```

Comment:
Could you extract this to own method? It's getting unreadable now.

Reply:
sure